### PR TITLE
Add Note about usage with SvelteKit to Readme

### PR DESCRIPTION
**Pull request by frederikhors**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,20 @@ npm i sswr
 yarn add sswr
 ```
 
+### Note about usage with SvelteKit
+
+Temporarily you need to add sswr to the Vite optimization exclusions in thevelve.config.js file:
```
Comment (benbender) [inline README.md]: should be `svelte.config.js`

File: README.md [hunk 2]
```diff
@@ -61,6 +61,20 @@ npm i sswr
 yarn add sswr
 ```
 
+### Note about usage with SvelteKit
+
+Temporarily you need to add sswr to the Vite optimization exclusions in the `svelte.config.js` file:
+
+```js
+vite: {
```
Comment (dominikg) [inline README.md:69]: kit key is missing here

File: README.md (hunk 1 above)
Comment (frederikhors) [reply README.md]: Is fixed already.

File: README.md (hunk 2 above)
Comment (frederikhors) [reply README.md]: Done!

**Comment by ConsoleTVs** [comment]
Thanks for the contributions to you as well @frederikhors !

**Pull request merged**